server/events.lua

```diff
@@ -202,3 +202,11 @@ RegisterNetEvent('QBCore:ToggleDuty', function()
     end
     TriggerClientEvent('QBCore:Client:SetDuty', src, player.PlayerData.job.onduty)
 end)
+
+RegisterServerEvent('qbx_core:server:onSetMetaData', function (meta, _, val, source)
```

Comment:
Why do we use this event instead of using the internal update metadata function?

Reply:
Hum true, idk if what i changed is what you meant for sure tho